tests/test_clients.py

```diff
@@ -19,5 +19,11 @@ def test_alter_fullscreen(hlwm):
     hlwm.call('attr clients.focus.fullscreen toggle')
     assert hlwm.get_attr('clients.focus.fullscreen') == 'false'
 
+
 def test_fullscreen_completion(hlwm):
     assert hlwm.complete("fullscreen") == 'false off on toggle true'.split(' ')
+
+
+def test_close_without_clients(hlwm):
+    # close may return HERBST_INVALID_ARGUMENT if there is no window to close
```

Comment:
But within the controlled environment of this test we should be able to *know* that there are no windows to close, right?

Shouldn't this be split into two tests like
 * `test_close_focused_window`
 * `test_close_nonexistent_focused_window`

Reply:
I've splitted the test.